### PR TITLE
Add stale action, remove stale bot

### PR DESCRIPTION
**Pull request by itsbagpack**
Converting current stale bot settings into a GitHub action that will run at 12 pm UTC every day.

### CODE REVIEW

File: .github/workflows/stale.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    steps:
+    - uses: actions/stale@v3
+      with:
+        stale-issue-message: >
+          This issue has been automatically marked as stale because it has not
+          had recent activity. It will be closed if no further activity occurs.
+          Thank you for your contributions.
+        stale-issue-label: "stale"
+        exempt-issue-labels: "pinned,security,translation"
+        days-before-issue-stale: 60
```
Comment (MikeMcQuaid) [inline .github/workflows/stale.yml:19]:
```suggestion
        days-before-issue-stale: 30
```

File: .github/workflows/stale.yml [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/stale@v3
+      with:
+        stale-issue-message: >
+          This issue has been automatically marked as stale because it has not
+          had recent activity. It will be closed if no further activity occurs.
+          Thank you for your contributions.
+        stale-issue-label: "stale"
+        exempt-issue-labels: "pinned,security,translation"
```
Comment (MikeMcQuaid) [inline .github/workflows/stale.yml:18]:
```suggestion
        exempt-issue-labels: "pinned,security"
```

**Review by MikeMcQuaid — APPROVED** [review]
Looks good! I'd suggest we reduce the number of days and exclude translations. I think after this we could also have/do the same for github/opensourcefriday and github/explore. Thoughts?

**Review by Daviontaylor — COMMENTED** [review]
Taylordavion87@gmail.com

**Review by Daviontaylor — CHANGES_REQUESTED** [review]
Taylordavion87@gmail.com

**Review by Anon79 — COMMENTED** [review]
ขอบคุณ

**Review by soffieswan043 — COMMENTED** [review]
#2534

**Comment by itsbagpack** [comment]
> I think after this we could also have/do the same for github/opensourcefriday and github/explore. Thoughts?

I think that's a solid plan @MikeMcQuaid

**Pull request merged**